spec/requests/users_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    it 'works! (now write some real specs)' do
+      get users_index_path
+      expect(response).to have_http_status(200)
+    end
+  end
+end
```

Comment:
- [ ] As stated in the requirements of this project, both the index and show actions should be tested. Spec files include:
```
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```

- Tests that ensure the correct view template is rendered
- Tests that ensure the response body includes the correct placeholder text
Let's kindly meet up with the requirements.

Reply:
well resolved! ✔️